### DISCUSSION TITLE
Connect to address localhost:25565 nevermind

### DISCUSSION THREAD

**Discussion by ERT4Wyatt** [mircokroon/minecraft-world-downloader#187] [Q&A]
I'm putting in the ip and and it won't connect, it keep trying to connect to localhost:25565 and not the IP I'm using?

**Comment by mircokroon** [comment]
Can you show a screenshot of the first tab, with the server address?

**Comment by oOctocatt** [comment]
> I'm putting in the ip and and it won't connect, it keep trying to connect to localhost:25565 and not the IP I'm using?

Hi! i'm having the same issue, could you explain how you fixed it please?

**Reply by ERT4Wyatt** [reply]
> > I'm putting in the ip and and it won't connect, it keep trying to connect to localhost:25565 and not the IP I'm using?
> 
> Hi! i'm having the same issue, could you explain how you fixed it please?

In minecraft use localhost:25565 and not the ip of the server.

**Reply by WhynotFulp** [reply]
Hi, can either of you share screen shots of how you were able to resolve this issue?
